components/AddYourProjectsGuide.tsx

```diff
@@ -91,58 +104,124 @@ const option2Steps = [
+    text: "Clone the repository",
+    code: "git clone https://github.com/<your_username>/ProjectsHut.git",
+    image: 
+      "https://user-images.githubusercontent.com/80768852/258655867-6c74b8f4-ee2f-48e8-8fc1-38679143175f.png"
   },
   {
     icon: <PiArrowFatLineRightFill size={18} color="white" />,
-    text: "Navigate to the project folder",
+    text: "Navigate to the ProjectsHut directory",
     code: "cd ProjectsHut",
+    image: 
+      "https://user-images.githubusercontent.com/80768852/258653517-fefecfe1-1f74-444b-8837-120c9f57cebf.png"
+  },
+  {
+    icon: <PiArrowFatLineRightFill size={18} color="white" />,
+    text: "Add a reference(remote) to the original repository",
+    code: "git remote add upstream https://github.com/priyankarpal/ProjectsHut.git",
+    image:
+      "https://user-images.githubusercontent.com/80768852/258653648-497133fd-c756-4065-9956-d2457b4ff53c.png"
+  },
+  {
+    icon: <PiArrowFatLineRightFill size={18} color="white" />,
+    text: "Check remotes for this repository",
+    code: "git remote -v",
+    image:
+      "https://user-images.githubusercontent.com/80768852/258653764-685d1dea-627c-4e4c-9444-c1ff05fb444e.png" 
+  },
+  {
+    icon: <PiArrowFatLineRightFill size={18} color="white" />,
+    text: "Always take a pull from the upstream repository to your main branch to keep it updated as per the main project repository",
+    code: "git pull upstream main",
+    image:
+      "https://user-images.githubusercontent.com/80768852/258656123-fea37c96-d16b-43e3-807d-76a2c01a9d4a.png"
   },
   {
     icon: <PiArrowFatLineRightFill size={18} color="white" />,
     text: "install dependencies",
     code: "pnpm i",
+    image:
+      "https://user-images.githubusercontent.com/80768852/258655966-ea8b5390-32d8-4f9e-8eb2-1eb840e9e7d1.png"
   },
   {
     icon: <PiArrowFatLineRightFill size={18} color="white" />,
-    text: "Create a new branch using your `GitHub Username`",
-    code: "git checkout -b [name_of_your_new_branch]",
+    text: "Create a new Branch",
+    code: "git checkout -b <your_branch_name>",
+    image: 
+      "https://user-images.githubusercontent.com/80768852/258654058-e0e8a964-56d1-4c33-91aa-0334771e8034.png"
   },
   {
     icon: <PiArrowFatLineRightFill size={18} color="white" />,
     text: "Run the project",
     code: "pnpm dev",
+    image:
+      "https://user-images.githubusercontent.com/80768852/258656216-24308a09-ae5a-4e7c-9c08-fbdd48e7f017.png"
   },
   {
     icon: <PiArrowFatLineRightFill size={18} color="white" />,
-    text: "Do Changes in Project you want.",
+    text: "Perform your desired Changes and if you are contributing a project then you can follow the below command",
     code: "",
   },
   {
     icon: <PiArrowFatLineRightFill size={18} color="white" />,
-    text: "Add your changes",
-    code: "git add .",
+    text: "Add following code to end of src/DB/projects.json",
+    code: codeString,
+  },
```

Comment:
remove the code. we don't need to add `json` code here

Reply:
It will be done by today

Reply:
![image](https://github.com/priyankarpal/ProjectsHut/assets/80768852/888bb017-be82-46e7-9f62-8fa436c5e6b2)

@priyankarpal while editing your highlighted issue is not seen. What do you think I should do now?

Reply:
Is it okay now @priyankarpal?